inference/cluster_store.py

```diff
@@ -24,12 +97,88 @@ def average_similarity(self):
         # TODO 
         pass
 
+    def euclidian_dist(self,entity1, entity2):
+        '''
+        :param entity1: X
+        :param entity2: Y
+        :return: euclidian distance between two entity vector (3D) X and Y
+        '''
+        X = self.original_data[entity1]
+        Y = self.original_data[entity2]
+        return math.sqrt(pow((X[0]-Y[0]),2) + pow((X[1]-Y[1]),2) + pow((X[2]-Y[2]),2))
```

Comment:
The length of an entity array will be arbitrary. Perhaps try this similar approach with a for loop.

Reply:
@RaihanSabique

Reply:
I didn't get it. You mean the length of entity_map and data are different?

Reply:
No, data can have arrays that are of length > 3. The current implementation is only finding distance based on the first three indices. For example:
```python
data = [
    [float, float, float, float],
    [float, float, float, float],
    [float, float, float, float]
]
entity_map = {
    0 : "name1",
    1 : "name2",
    2 : "name3"
}
c_ob= ClusterStore(data,entity_map)
```

In this case, the last index will be ignored when calculating distance.

Reply:
got it